File: c#/Loop.cs
using System;

namespace Loop
{
    class Loop
    {
        static void Main(string[] args)
        {
            Console.WriteLine("Number N and x input=");
            String[] Nx = Console.ReadLine().Split(" ");
            int N = int.Parse(Nx[0]);
            int x = int.Parse(Nx[1]);
            
            N = int.Parse(Console.ReadLine());
            x = int.Parse(Console.ReadLine());
            
            
            for (int i = N; i <= x; i++) {
                Console.Write(i + ".)");
                if (i % 2 == 0) {
                  Console.Write(" " + i + " is even");
                } else {
                  Console.Write(" " + i + " is odd");
                }
                
                if (i % 3 == 0) {
                  Console.Write(", can be divide by 3");
                } else {
                  Console.Write(", can't be divide by 3");
                }
                
                if (i % 5 == 0) {
                  Console.Write(", can be divide by 5");
                } else {
                  Console.Write(", can't be divide by 5");
                }
                
                Console.Write("\n");
            }
            
            int i = x;
            do {
                Console.WriteLine(i);
                
                i--;
            } while (i > N);
        }
    }
}

File: c/loop.c
#include <stdio.h>

int main() {
  int N;
  int x;
  printf("Number N input=");
  scanf("%d", &N);
  printf("Number x input=");
  scanf("%d", &x);
  
  for (int i = N; i <= x; i++) {
    printf("%d.)", i);
    if (i % 2 == 0) {
      printf(" %d is even", i);
    } else {
      printf(" %d is odd", i);
    }
    
    if (i % 3 == 0) {
      printf(", can be divide by 3");
    } else {
      printf(", can't be divide by 3");
    }
    
    if (i % 5 == 0) {
      printf(", can be divide by 5");
    } else {
      printf(", can't be divide by 5");
    }
    
    printf("\n");
  }
  
  int i = x;
  do {
    printf("%d\n", i);
    
    i--;
  } while (i > N);
  
  return 0;
}
